src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The href attribute in the link tag should point to a CSS file, not a SCSS file. SCSS files need to be compiled to CSS before they can be linked in HTML.